src/debugging/netSdk/NetSdkDebugHelper.ts

```diff
@@ -56,66 +55,53 @@ export class NetSdkDebugHelper extends NetCoreDebugHelper {
     }
 
     public async afterResolveDebugConfiguration(context: DockerDebugContext, debugConfiguration: DockerDebugConfiguration): Promise<void> {
-        const { task, promise } = netSdkRunTaskProvider.createNetSdkRunTask(
-            {
-                netCore: {
-                    appProject: await this.inferProjPath(context.actionContext, context.folder),
-                }
+        const projectInfo = await this.getProjectInfo(debugConfiguration);
+        const runDefinition: Omit<NetSdkRunTaskDefinition, "type"> = {
+            netCore: {
+                appProject: await this.inferProjPath(undefined, debugConfiguration.netCore),
```

Comment:
This code is implicitly relying on the fact that debugConfiguration.netCore.appProject will always be set; otherwise this.inferProjPath would throw an exception due to context being null here. Better to just directly set the value from the configuration.

```suggestion
                appProject: debugConfiguration.netCore.appProject,
```